src/message/messageActionSheet.js

```diff
@@ -73,6 +73,9 @@ type AuthMessageAndNarrow = {
   narrow: [],
 };
 
+const isAnOutboxMessage = ({ message }: Message): boolean =>
+  message.isOutbox && message.isOutbox === true;
```

Comment:
`message.isOutbox === true' will never execute, as ` message.isOutbox` is already true. In that case, though, you totally do not need this extracted as a function.